commit.go

```diff
@@ -278,9 +281,18 @@ func (b *Builder) Commit(ctx context.Context, dest types.ImageReference, options
 	}()
 
 	// Check if the commit is blocked by $BUILDER_REGISTRY_SOURCES.
-	if err := checkRegistrySourcesAllows("commit to", dest); err != nil {
+	insecure, err := checkRegistrySourcesAllows("commit to", dest)
+	if err != nil {
 		return imgID, nil, "", err
 	}
+	if insecure {
+		if systemContext.DockerInsecureSkipTLSVerify == types.OptionalBoolFalse {
+			return imgID, nil, "", errors.Errorf("can't require tls verify on insecurely allowed registry")
```

Comment:
```suggestion
			return imgID, nil, "", errors.Errorf("can't require tls verification on an insecured registry")
```

Reply:
It doesn't look like we document in the man pages.  We might want to add something like: "This option cannot be used when talking to an insecure registry." in the verbiage for the tls-verify option for each of these.

Reply:
Change to `TLS verification cannot be used when talking to an insecure registry.`, since the option --tls-verify=false is allowed if using insecure registry.